orders/views.py

```diff
@@ -1,3 +1,72 @@
-from django.shortcuts import render
+import json
+
+from django.http.response   import JsonResponse
+from django.views           import View
+
+from .models                import Cart
+from FRA_WE_BACK.core.utils import log_in_decoratorr
+
+
+class CartView(View):
+    @log_in_decoratorr
+    def get(self, request):
+            carts  = Cart.objects.filter(user = request.user)
+            cart_items = []
+            for cart in carts:
+                cart_items = [{
+                    "product_id"      : cart.product.id,
+                    "product_kr_name" : cart.product.kr_name,
+                    "product_en_name" : cart.product.en_name,
+                    "product_price"   : cart.product.price,
+                    "count"           : cart.count,
+                    "product_image"   : [image.url for image in cart.product.image_set.all()]
+                }]
+            return JsonResponse({'cart_items' : cart_items}, status=200)
+
+    @log_in_decoratorr
+    def post(self, request):
+        try:
+            data               = json.loads(request.body)
+            Cart.objects.create(
+                product_id     = data['product_id'],
+                user           = request.user,
+                count          = data['count']
+            )
+            return JsonResponse({'message' : 'SUCCESS'}, status=201)
```

Comment:
status code 201은 인증이 성공했을 때의 코드입니다. 성공 메세지에 맞게 수정해주세요.
```suggestion
            return JsonResponse({'message' : 'SUCCESS'}, status=200)
```